logging/tests/unit/test__http.py

```diff
@@ -476,6 +476,26 @@ def test_sink_create_ok(self):
         self.assertEqual(conn._called_with['path'], path)
         self.assertEqual(conn._called_with['data'], SENT)
 
+    def test_sink_create_unique_writer_identity(self):
+        SENT = {
+            'name': self.SINK_NAME,
+            'filter': self.FILTER,
+            'destination': self.DESTINATION_URI,
+        }
+        
+        conn = _Connection({})
+        client = _Client(conn)
+        api = self._make_one(client)
+
+        api.sink_create(
+            self.PROJECT, self.SINK_NAME, self.FILTER, self.DESTINATION_URI, True)
+
+        self.assertEqual(conn._called_with['method'], 'POST')
+        path = '/projects/%s/sinks' % (self.PROJECT,)
+        self.assertEqual(conn._called_with['path'], path)
+        self.assertEqual(conn._called_with['data'], SENT)
+        self.assertEqual(conn._called_with['query_params'], {'uniqueWriterIdentity': True})
```

Comment:
If `conn._called_with` is a dictionary, why not just construct the expected dictionary and assert equality? In the current approach, `_called_with` could grow new keys and your test wouldn't detect that.